### DISCUSSION TITLE
Waveshare print-able mount

### DISCUSSION THREAD

**Discussion by Xorlent** [Westcott1/Waveshare-ESP32-S3-Touch-LCD-4.3-and-Arduino#7] [Show and tell]
Love this repo and thank you for making this available @Westcott1!  I've built a printable mount to support this project for anyone interested.  I also linked to your project in the README for folks looking to develop with this Waveshare device.  
  
[1-Gang-Waveshare-ESP32-Touch](https://github.com/Xorlent/1-Gang-Waveshare-ESP32-Touch)

**Comment by Westcott1** [comment]
Hi Xorlent,
Thanks for your kind words, and the link back to this repo.
